public/app/features/scopes/tests/viewMode.test.ts

```diff
@@ -0,0 +1,73 @@
+import { config } from '@grafana/runtime';
+import { DashboardScene } from 'app/features/dashboard-scene/scene/DashboardScene';
+
+import { scopesDashboardsScene, scopesSelectorScene } from '../instance';
+
+import { enterEditMode, openSelector, toggleDashboards } from './utils/actions';
+import { expectDashboardsClosed, expectDashboardsDisabled, expectScopesSelectorClosed } from './utils/assertions';
+import { getDatasource, getInstanceSettings, getMock } from './utils/mocks';
+import { renderDashboard, resetScenes } from './utils/render';
+
+jest.mock('@grafana/scenes', () => ({
+  __esModule: true,
+  ...jest.requireActual('@grafana/scenes'),
+  sceneUtils: {
+    ...jest.requireActual('@grafana/scenes').sceneUtils,
+    registerVariableMacro: () => () => undefined,
```

Comment:
why is this mocked?

Reply:
I'm using `transformSaveModelToScene` to get the same scene that we use for dashboards.

`transformSaveModelToScene` is calling `registerDashboardMacro` which causes tests to fail due to calling the macro registration twice.

Reply:
@bfmatei no it's not, activating a dashboard is calling registerDashboardMacro (as it's a behavior), another way to fix it is just to deactivate the scene after each test, I think that is how most other tests handle this, but I guess mocking also works :)

Reply:
Hmm, let me have a look again over this but I do remember receiving `variable macro registered twice` error w/o mocking it.

I tried cleaning up the render after each test using Jest's `cleanup` but that didn't work out. Is there another way to deactivate a scene?

Reply:
Hm, I removed the mock and the problem is not there anymore. I removed the unnecessary mocks.